java/ql/lib/ext/org.geogebra.web.full.main.model.yml

```diff
@@ -0,0 +1,7 @@
+extensions:
+
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["org.geogebra.web.full.main", "FileManager", True, "open", "(String,String)", "", "Argument[0]", "open-url", "ai-generated"]
```

Comment:
This appears to be redirecting a browser instead of opening a server-side connection:

```suggestion
      - ["org.geogebra.web.full.main", "FileManager", True, "open", "(String,String)", "", "Argument[0]", "url-redirect", "ai-generated"]
```